tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Lambda/LambdaCommon.cs

```diff
@@ -32,8 +32,17 @@ internal static Scope CreatePlaceholderScope(Tracer tracer, string traceId, stri
         }
         else
         {
-            Log($"creating the placeholder traceId = {traceId}");
-            span = tracer.StartSpan(PlaceholderOperationName, tags: null, serviceName: PlaceholderServiceName, traceId: (TraceId)Convert.ToUInt64(traceId), addToTraceContext: false);
+            if (traceIdUpper64 == 0)
+            {
+                Log($"creating the placeholder traceId = {traceId}");
+                span = tracer.StartSpan(PlaceholderOperationName, tags: null, serviceName: PlaceholderServiceName, traceId: (TraceId)Convert.ToUInt64(traceId), addToTraceContext: false);
+            }
+            else
+            {
+                var traceIdLower64 = Convert.ToUInt64(traceId);
+                Log($"creating the placeholder traceId = {traceIdUpper64}{traceIdLower64}");
```

Comment:
`{traceIdUpper64}{traceIdLower64}` will result in the base-10 representation of these two values side by side. We never look at 128-bit ids that way. We use hexadecimal format instead.